openedx/features/teams/urls.py

```diff
@@ -10,6 +10,7 @@
 
     url(r"^browse_teams/$", browse_teams, name="teams_dashboard"),
     url(r"^browse_teams/" + TOPIC_ID_PATTERN + "/$", browse_topic_teams, name="browse_topic_teams"),
+    url(r"^create/$", create_team, name="create_team_without_topic"),
```

Comment:
let's re-name it to something more appropriate since it also creates a team with topic